### PR TITLE
Added my logo design to website

### PR DESCRIPTION
**Pull request by Dodecane**

### CODE REVIEW

File: .coafile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [default]
-bears = SpaceConsistencyBear, LineLengthBear
```
Comment (hemantjadon) [inline .coafile:2]: Why removed SpaceConsistencyBear. It is an important part of linting.

**Review by hemantjadon — CHANGES_REQUESTED** [review]

**Comment by Dodecane** [comment]
Done

**Comment by Princu7** [comment]
@LeeCareGene Great. But there are some additional works which you have to do

*  Always give the link(if any) of the task that you are completing. Also, post your gh-pages link for easier and faster reviewing.

*  Squash the commits into one. There have been many links posted on FossAsia #gci channel on slack on how to squash the commits. Please go through them.
Cheers!!

**Comment by Dodecane** [comment]
https://codein.withgoogle.com/dashboard/task-instances/4811085861355520/
leecaregene.github.io

**Review by hemantjadon — APPROVED** [review]

**Review by hemantjadon — CHANGES_REQUESTED** [review]
Please squash the commits into one. as mentioned by @Princu7

**Comment by Dodecane** [comment]
I didn't close this ._.

**Comment by abishekvashok** [comment]
hey what happened @LeeCareGene  who deleted your commits?

**Comment by hemantjadon** [comment]
@LeeCareGene 
It seems that you have deleted your branch by accident

**Comment by Dodecane** [comment]
nvm undoed it

**Comment by abishekvashok** [comment]
@LeeCareGene again, you messed up try reverting it again... :)

**Comment by Dodecane** [comment]
Yay did it

**Comment by abishekvashok** [comment]
@LeeCareGene well done can you share your code! and give a preview or screenshot to your code so that mentors can judge it quickly!

**Comment by Princu7** [comment]
@LeeCareGene Congrats. You have shown great dedication. Keep up the good work.!! You have to submit the task in order for me to approve it. Please submit the task!!

**Review by hemantjadon — APPROVED** [review]

**Comment by jigyasa-grover** [comment]
@LeeCareGene Finally, great persistence :)

**Comment by hemantjadon** [comment]
Hey @LeeCareGene 
Nice commitment, Keep It up :)

Submit your task for review (after completing other parts of it) So we can approve it. 

Cheers :)

**Pull request merged**